example/lib/pages/single_world_polys.dart

```diff
@@ -0,0 +1,250 @@
+                        .toList(),
+                    color: const Color(0xFF000000),
+                  ),
+                ],
+              ),
+              PolylineLayer(
+                simplificationTolerance: 0,
+                drawInSingleWorld: false,
+                polylines: [
+                  Polyline(
+                    points: _polylinePoints
+                        .map((latLng) =>
+                            LatLng(latLng.latitude + 25, latLng.longitude))
+                        .toList(),
+                    strokeWidth: 8,
+                    color: const Color(0xFFFF0000),
+                    pattern: const StrokePattern.dotted(),
+                  ),
+                  Polyline(
+                    points: const [
+                      LatLng(-80, 150),
+                      LatLng(-80, -170),
+                      LatLng(-75, -170),
+                      LatLng(-75, 150),
+                      LatLng(-80, 150),
+                    ],
+                    strokeWidth: 8,
+                    color: Colors.yellow,
+                  ),
+                ],
+              ),
+              PolylineLayer(
+                simplificationTolerance: 0,
+                drawInSingleWorld: true,
+                polylines: [
+                  Polyline(
+                    points: _polylinePoints
+                        .map((latLng) =>
+                            LatLng(latLng.latitude + 0, latLng.longitude))
+                        .toList(),
+                    strokeWidth: 8,
+                    color: Colors.red,
+                    pattern: StrokePattern.dashed(segments: [50, 20]),
+                  ),
+                  Polyline(
+                    points: const [
+                      LatLng(80, 150),
+                      LatLng(80, -170),
+                      LatLng(75, -170),
+                      LatLng(75, 150),
+                      LatLng(80, 150),
+                    ],
+                    strokeWidth: 8,
+                    color: Colors.yellow,
+                  ),
+                ],
+              ), /*PolygonLayer(
+                drawInSingleWorld: true,
+                polygons: [
+                  Polygon(
+                    points: [
+                      const LatLng(90, -180),
+                      const LatLng(90, 180),
+                      const LatLng(-90, 180),
+                      const LatLng(-90, -180),
+                    ],
+                    color: Colors.amber,
+                    borderColor: Colors.black,
+                    borderStrokeWidth: 5,
+                    holePointsList: [
+                      [
+                        LatLng(46, -9),
+                        LatLng(46, -8),
+                        LatLng(45.5, -7.5),
+                        LatLng(45, -8),
+                        LatLng(45, -9),
+                      ]
+                    ],
+                  ),
+                ],
+              ),*/
```

Comment:
This demonstrates a question relating to the recently added `invertedFill`: is it still necessary. Is it now possible to reliably achieve everything we did before the multi-world change, and/or with the feature set introduced by `invertedFill`, or is there still some advantage to keeping `invertedFill`?

Wdyt?

Reply:
@JaffaKetchup `invertedFill` is still necessary if you want to display polylines as transparent holes.
![image](https://github.com/user-attachments/assets/ecdec0d1-b797-4315-b038-a71a3aeaeae3)